.storybook/main.js

```diff
@@ -42,24 +42,48 @@ const config = {
   },
   // index markdown stories so they can be part of the generated menu and lazy loaded
   experimental_indexers: async (existingIndexers) => {
-    return [markdownIndexer, ...existingIndexers]
+    return [markdownIndexer, ...existingIndexers ?? []]
   },
-  viteFinal (config, { configType }) {
-    // transform markdown files to CSF so they can be loaded by storybook
-    config.plugins?.unshift(rollupMdToCsfPlugin());
+  async viteFinal (config, { configType }) {
+    const { mergeConfig } = await import('vite');
```

Comment:
**note:** I don't understand why we **have** to use a dynamic import but this is what the docs do (see [viteFinal - Storybook](https://storybook.js.org/docs/api/main-config-vite-final#options).

If you try to use a static import instead, vite throws a warning about the fact that you're using the `CJS` version of vite's API:
`The CJS build of Vite's Node API is deprecated.`